### ISSUE TITLE
AttributeError: 'Dataset' object has no attribute 'Count'

### ISSUE THREAD

**Issue by gsakkis** [pyviz-topics/examples#144]
I'm running locally the [NYC taxi](https://github.com/pyviz-topics/examples/blob/master/nyc_taxi/nyc_taxi.ipynb) notebook and getting the error in the subject at the "Customizing datashader" section.

versions:
- holoviews: 1.14.3
- datashader: 0.12.1

**Comment by jbednar** [comment]
I've checked, and if you follow the examples.pyviz.org user guide and do "anaconda-project run" (and specifically "anaconda-project run notebooks", to get the nyc_taxi notebook), everything works as designed, using pinned versions holoviews=1.12.3, datashader=0.7, and xarray=0.14.1. So in some sense there's no issue here; examples.pyviz.org explicitly only supports the pinned versions, which you obtain by running examples as described in the user guide.

That said, I was also able to reproduce this problem when I ran the code with the most recent holoviews, datashader, and other library versions.  From the error message I'd suspect something to do with xarray has changed that we need to adapt to, but I was not able to verify that. The problem is present in the latest xarray release (0.17), but I also found the same issue with xarray 0.16 and 0.15. The problem went away with xarray 0.14.1, but installing xarray 0.14.1 downgraded holoviews, datashader, panel, etc. to the already-known-working pinned versions, and so I can't tell whether the problem was introduced in xarray 0.15 or whether it was e.g. due to unrelated changes in datashader since 0.7.

So I'll leave this issue open until we can figure out what the problem with newer xarray and/or datashader versions is.
